parsing/lexer.mll

```diff
@@ -124,12 +124,15 @@ let keyword_edition v =
 
 let parse_keyword_edition s =
   let parse_version s =
+  let bad_version () =
+    raise (Arg.Bad "Ill-formed version in keywords flag,\n\
+                    the expected format is %d.%d .")
```

Comment:
It took me a while to understand that `%d.%d` is not a format literal here, it is something you would show to the user. I would rather use: `is <major>.<minor>, for example 4.12`.

Reply:
Note: I would use "the supported format", in case some future compiler releases starts accepting more (or less) formats. Maybe the format provided by the user is not wrong (for the up-to-date version of the compiler), we just don't support it yet.